services/movie.py

```diff
@@ -1,11 +1,14 @@
 from django.db.models import QuerySet
+from django.db import transaction
+
```

Comment:
No need to have this line

Reply:
it's the same here
services\movie.py:11:6: F821 undefined name 'QuerySet'